kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/usagelimits/cooldowns/DefaultCooldownHandler.kt

```diff
@@ -0,0 +1,206 @@
+        type: CooldownType,
+    ): String {
+        val locale = context.locale()
+        val translationsProvider = context.event.command.translationsProvider
+        val commandName = context.event.command.getFullName(locale)
+        return when (type) {
+            CachedUsageLimitType.COMMAND_USER -> translationsProvider.translate(
+                "cooldown.notifier.commandUser",
+                locale,
+                replacements = arrayOf(discordTimeStamp, commandName)
+            )
+
+            CachedUsageLimitType.COMMAND_USER_CHANNEL -> translationsProvider.translate(
+                "cooldown.notifier.commandUserChannel",
+                locale,
+                replacements = arrayOf(discordTimeStamp, commandName, context.channel.mention)
+            )
+
+            CachedUsageLimitType.COMMAND_USER_GUILD -> translationsProvider.translate(
+                "cooldown.notifier.commandUserGuild",
+                locale,
+                replacements = arrayOf(discordTimeStamp, commandName)
+            )
+
+            CachedUsageLimitType.GLOBAL_USER -> translationsProvider.translate(
+                "cooldown.notifier.globalUser",
+                locale,
+                replacements = arrayOf(discordTimeStamp)
+            )
+
+            CachedUsageLimitType.GLOBAL_USER_CHANNEL -> translationsProvider.translate(
+                "cooldown.notifier.globalUserChannel",
+                locale,
+                replacements = arrayOf(discordTimeStamp, context.channel.mention)
+            )
+
+            CachedUsageLimitType.GLOBAL_USER_GUILD -> translationsProvider.translate(
+                "cooldown.notifier.globalUserGuild",
+                locale,
+                replacements = arrayOf(discordTimeStamp)
+            )
+
+            CachedUsageLimitType.GLOBAL -> translationsProvider.translate(
+                "cooldown.notifier.global",
+                locale,
+                replacements = arrayOf(discordTimeStamp)
+            )
+
+            CachedUsageLimitType.GLOBAL_CHANNEL -> translationsProvider.translate(
+                "cooldown.notifier.globalChannel",
+                locale,
+                replacements = arrayOf(discordTimeStamp, commandName)
+            )
+
+            CachedUsageLimitType.GLOBAL_GUILD -> translationsProvider.translate(
+                "cooldown.notifier.globalGuild",
+                locale,
+                replacements = arrayOf(discordTimeStamp)
+            )
+
+            else -> translationsProvider.translate(
+                "cooldown.notifier.generic",
+                locale,
+                replacements = arrayOf(type.toString().lowercase())
+            )
+        }
+    }
+
+    override suspend fun onExecCooldownUpdate(
+        commandContext: CommandContext,
+        context: DiscriminatingContext,
+        success: Boolean,
+    ) {
+        if (!success) return
+        for (t in CachedUsageLimitType.values()) {
```

Comment:
Missing braces and a newline here.

Also, `t` and `u` are not acceptable variable names, it's not at all clear what they're meant to be.